src/components/Tooltip/Tooltip.tsx

```diff
@@ -28,10 +28,10 @@ const Tooltip = ({
   isOpen,
   setIsOpen,
 }: ITooltip) => {
-  const tooltipRef = createRef()
-  const tooltipArrowRef = createRef()
-  const tooltipShowDelayTimerRef = useRef<ReturnType<typeof setTimeout>>()
-  const tooltipHideDelayTimerRef = useRef<ReturnType<typeof setTimeout>>()
+  const tooltipRef = useRef<HTMLElement>(null)
+  const tooltipArrowRef = useRef<HTMLDivElement>(null)
```

Comment:
was `createRef()` being used intentionally? didn't seem like it. `useRef()` is preferred when using functional components

Reply:
yeah, you're right!

can you do the change and test if the behavior still as expected, please?